### DISCUSSION TITLE
Gradient transition

### DISCUSSION THREAD

**Discussion by moishinetzer** [tailwindlabs/tailwindcss#4145] [Help] [answered]
Heya! Love tailwind, it's how I learned CSS and makes it much more enjoyable and easier to use as a beginner.

Is there a way to have a gradients "from" or "to" colors change by transition on hover for example? 

(Also gradients look gorgeous in general big fan)

**Comment by simonswiss** [comment, marked as answer]
Hey!

Unfortunately, as far as I know, it's still not possible to transition a `linear-gradient`, which is what Tailwind uses under the hood for the background gradients.

A "workaround" way to achieve this is to have an "overlay" gradient with different colors, and animate the opacity of that overlay on hover.

Here's an example of that in a Tailwind Play (and video):

https://play.tailwindcss.com/2dvsucdRwo


https://user-images.githubusercontent.com/485747/115818586-68ea9b80-a440-11eb-9224-03a30c549488.mov

**Reply by moishinetzer** [reply]
Wow, this is a great workaround thanks very much! Loving tailwind and all the efforts put in to it.
